content/blobstore-tls.md

```diff
@@ -0,0 +1,83 @@
+!!! note
+    Applicable for director version 266.2.0+
+
+    Linux stemcells v3586.5+ (agent 2.83.0)
+
+    Windows 2016 stemcell v1709.9+, Windows 2012R2 v1200.20+ (agent 2.110.0)
+
+# Rotating the Blobstore CA
+
+Applies to a director with TLS enabled for the default DAV blobstore.
+
+Example director configuration enabling TLS for the DAV blobstore. Refer to [bosh-deployment](https://github.com/cloudfoundry/bosh-deployment) for a full manifest reference.
+```
+...
+instance_groups:
+- name: bosh
+  properties:
+    agent:
+      env:
+        bosh:
+          blobstores:
+          - provider: dav
+            options:
+              endpoint: https://((internal_ip)):25250
+              user: agent
+              password: ((blobstore_agent_password))
+              tls:
+                cert:
+                  ca: ((blobstore_ca.certificate))
+
+    blobstore:
+      address: ((internal_ip))
+      port: 25250
+      provider: dav
+      director:
+        user: director
+        password: ((blobstore_director_password))
+      agent:
+        user: agent
+        password: ((blobstore_agent_password))
+      tls:
+        cert:
+          ca: ((blobstore_ca.certificate))
+          certificate: ((blobstore_server_tls.certificate))
+          private_key: ((blobstore_server_tls.private_key))
+...
+```
+
+
+## Ignored or Stopped Instances
+
+Take note of any ignored instances with `bosh instances --details`. These instances will not be considered when redeploying or recreating instances. Ignored instances will only receive new certificates when recreated.
+
+Any instances stopped with `bosh stop` will not receive the new CA or certificate until recreated. Performing `bosh start` on these instances will transition the VM to running successfully. However, if a redeploy does not trigger a recreate, this instance will fail to update. The agent consults the blobstore for templates during VM lifecycle changes.
+
+Instances that have been stopped with `bosh stop --hard` have been destroyed, keeping the persistent disk if applicable. Performing a `bosh start` on those instances will recreate the VM. These instances will then receive the new CA.
+
+## Introducing a New Certificate Authority (CA)
```

Comment:
@h4xnoodle the steps specified can introduce a situation where deployments will fail (as you have mentioned in the "Troubleshooting" section).

Let's say an operator performs the steps here, and then before attempting a recreate of all the VMs, a critical deployment need to happen.
This deployment will fail unless a VM recreation is performed, which can be very costly from time perspective, especially in production environments.
I think we should have a list of steps for blobstore cert rotation that is more immune to these situations and more stable; where we probably need to concatenate the old and new CAs together.

wdyt ?

Reply:
My first thoughts are who is the audience of this documentation and if we are to describe every single scenario that someone could have vs the 'happy path' guide. I think we could add a note, and describe what a work-around is in the troubleshooting section. 

Ultimately I think we should be able to eventually rotate without all these steps if we can send the new CAs/process them over NATS.

Reply:
+1 to what Jamil said.  You can't download logs, run colocated errands, or update code on VMs with blobstore CAs that don't match the director.